### ISSUE TITLE
Fragment support for latex environments

### ISSUE THREAD

**Issue by podiki** [yjwen/org-reveal#181]
Thanks for this great org-mode extension. It does not seem that fragment support is working or enabled for latex environments. The exported html has just the plain text for the equation environment, which is then parsed by MathJax to output the formula correctly. This is a small example; the slide appears all at once.

```
* Test equation
#+ATTR_REVEAL: :frag roll-in
\begin{equation}
e^{-i\pi} = -1
\end{equation}
```

**Comment by podiki** [comment]
It looks like the solution is to wrap the output equation environment in a `<div class="fragment roll-in"></div>`, but I'm not sure where exactly this needs to happen in ox-reveal. Should be a simple fix.

**Comment by yjwen** [comment]
The issue has been fixed by merge #195 .
Thanks for @acowley 's efforts again.
